content/en/tracing/setup_overview/custom_instrumentation/ruby.md

```diff
@@ -238,7 +235,7 @@ get '/posts' do
     span.set_tag('posts.count', @posts.length)
 
     # Trace the template rendering
-    Datadog.tracer.trace('template.render') do
+    Datadog::Tracing.trace('template.render') do
       erb :index
     end
   end
```

Comment:
⬇️  Below in the post-processing traces section there's several references to `Datadog::Pipeline` which are outdated (should be `Datadog::Tracing` instead, if I'm not mistaken)

Reply:
Updated these!